<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="image_backdop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>backdop3.jpg</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="welcome_frame">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>311</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Himalaya</family>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius: 40px;
border: 0.5px solid white;

background-color: qlineargradient(spread:pad, x1:0.642, y1:1, x2:0.542, y2:0, stop:0.121053 rgba(81, 31, 82, 255), stop:0.252632 rgba(105, 59, 105, 255), stop:0.463158 rgba(213, 136, 125, 255), stop:0.678947 rgba(169, 135, 168, 255), stop:0.710526 rgba(169, 135, 168, 255), stop:0.947368 rgba(48, 63, 159, 255));
background-color: rgba(80, 68, 111, 0.3);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color : white;
border: transparent;

</string>
    </property>
    <property name="text">
     <string>welcome</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_but">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>50</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ border-radius: 20px;
 background-color: #80669d;
color : white;
border: 0.5px solid white; 
}

QPushButton:pressed{
font: 8pt &quot;Sylfaen&quot;;
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_but">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>50</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ border-radius: 20px;
 background-color: #80669d;
color : white;
border: 0.5px solid white; 
}

QPushButton:pressed{
font: 8pt &quot;Sylfaen&quot;;
}</string>
    </property>
    <property name="text">
     <string>SIGNUP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color : white;
border: transparent;

</string>
    </property>
    <property name="text">
     <string>create a new account or login</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/boomh/Downloads/resource.qrc"/>
 </resources>
 <connections/>
</ui>
